### ISSUE TITLE
[Issue] Songs cut of before the end

### ISSUE THREAD

**Issue by Sybbbb** [JasonZigelbaum/jqbx-issues#94]
Sometimes (what feels to me like 20 - 30% of the time) a song cuts off before the end. Usually around 30 secs before the end of a song. It automatically goes on to the next song.

**Comment by Roanmh** [comment]
I have also had this problem, bit only in the web app. The OS X client does not seem to do so.

**Comment by HollandOates** [comment]
No, it's happening to me on the OSX client. Three songs in a row now.

**Comment by expandstudios** [comment]
I also have this problem fairly consistently.
Web app playing in Chrome.

I have a room with a colleague and they also experience the problem from there end, seems our sound is in sync as we are both having the songs cut off early. Sometimes it can be as early as half way through a song and other times it is nearer to the end. Doesn't happen ALL the time but I would agree with 20-30%.

**Comment by navels** [comment]
Just starting using JQBX and can confirm this is happening on the Mac client.  Pretty annoying.
